quiz.rb

```diff
@@ -14,24 +14,33 @@
 # Take a string and return an array of normalized words.
 # normalized words contain no punctuation and are all lower case
 def normalize_words words
+  paragraph.each_char.downcase.split/\W+/
```

Comment:
the each_char enumerator is breaking your code here. theres no need to introduce and enumerator